week2/The system of the student's score management. v1.0.cpp

```diff
@@ -0,0 +1,160 @@
+#include <stdio.h>
+
+void readscore(int n, long student_number[], int score[]);
+void print_total_and_average(int n, int score[]);
+void rank_by_grade_in_descending_order(int n, long student_number[], int score[]);
+void print_score(int n, long student_number[], int score[]);
+void rank_by_student_number_in_ascending_order(int n, long student_number[], int score[]);
+int search_student_rank_and_grade_by_student_number(int n, long student_number[], int score[]);
+void count_the_percentage_of_the_each_category(int n, long student_number[], int score[]);
+
+int main()
+{
+	long student_number[30];
+	int score[30], n, _switch;
+	printf("1.Input record\n2.Caculate total and average score of course\n3.Sort in descending order by score\n");
+	printf("4.Sort in ascending order by number\n5.search by number\n6.Statistic analysis\n0.Exit\n\n");
+	do {
+		printf("Please enter your choice:");
+		scanf_s("%d", &_switch);
+		switch (_switch)
+		{
+			case 1:

```

Comment:
For `case` label, instead of using raw number directly, a common practice is to claim an `enum` or macro-definition. See [here](https://github.com/torvalds/linux/blob/6cfae0c26b21dce323fe8799b66cf4bc996e3565/drivers/usb/mtu3/mtu3_dr.c#L164-L186) for example.